pkg/controllers/scheduler/planner/planner_test.go

```diff
@@ -429,6 +430,22 @@ func TestWithExisting(t *testing.T) {
 		},
 	)
 
+	doCheckWithExisting(t,
+		&testCase{
```

Comment:
Let's add one more test case where `limitedCapacity` is not 0

Reply:
Added.